.github/workflows/run_backend_services_tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Backend Services Tests
+# targeting pull requests to master branch
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  coverage:
+    permissions:
+      checks: write
+      pull-requests: write
+      contents: write
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [18.x]
+        service: [user-service, question-service]
+    steps:
+      - name: Checkout Current Branch
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{matrix.node-version}}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{matrix.node-version}}
+
+      - name: Clean up
+        run: |
+          rm -rf backend/${{matrix.service}}/node_modules
+
+      - name: Install Dependencies
+        run: |
+          cd backend/${{matrix.service}}
+          npm ci
+
+      - uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+
+      - name: Run Tests and Collect Coverage Reports
```

Comment:
Correct me if im wrong but does this fill the PR with reports every commit/push to the branch while the PR is active?

![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/assets/70256674/47421c37-c6a5-4f8d-aab5-6b0aaade308a)

So if we have like 4-6 microservices won't these generate a lot of spam/comments in our PRs 😅 Personally I dont think we need to generate the reports bah cos we can see it in the checks below:

![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/assets/70256674/4378f89e-2677-4202-8fc2-c439fd65784a)

Reply:
Yea, according to the current set up. If we have 6 microservices eventually, then there will be 6 comments attached for every PR we created haha. Alright, I will delete the PR comment feature.